app/main.py

```diff
@@ -1,3 +1,61 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance(km={self.km})"
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if type(other) == int or type(other) == float:
```

Comment:
Use `isinstance` to check the variable type. It's important in the case of inheritance that will be discussed in detail on the OOP topic